app/ClientSrc/vue/TrialTimeSelect/ChooseAvailability.vue

```diff
@@ -1,9 +1,328 @@
+              <i class="fas fa-trash" />
+            </button>
+          </div>
+          <div class="alert alert-warning" v-if="formattedSelection.length === 0">
+            <i class="fa fa-exclamation-triangle" />
+            Choose at least one starting date for your trial.
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script>
+import { formatDate } from "./helpers.js";
+
 export default {
   name: "ChooseAvailability",
+
+  data: () => ({
+    // show or hide extra "dates-info" text
+    showInfo: false,
+
+    selected: [],
+  }),
+
+  props: {
+    dates: {
+      type: Array,
+      default: () => [],
+    },
+
+    trialLength: {
+      type: Number,
+      default: 0,
+    },
+  },
+
+  computed: {
+    formattedDates() {
+      return this.dates.map(formatDate);
+    },
+
+    formattedSelection() {
+      return this.selected.map(formatDate);
+    },
+
+    visibleDates() {
+      return this.formattedDates.slice(0, 8).filter((date) => {
+        return !this.selected.includes(date.isoDate);
+      }); // temporary
+    },
+  },
+
+  // expand "Key Dates" info box by default on larger screens
+  mounted() {
+    if (window.innerWidth >= 768) {
+      this.showInfo = true;
+    }
+  },
+
+  methods: {
+    dateClasses(date) {
+      return {
+        selected: this.selected.includes(date.isoDate),
+      };
+    },
+
+    /**
+     * Removes a date from the selection.
+     *
+     * @param {string} isoDate - date string to remove
+     */
+    unselect(isoDate) {
+      this.selected = this.selected.filter((date) => date !== isoDate);
+    },
+  },
 };
 </script>
+
+<style scoped lang="scss">
+@import "../../sass/variables";
+
+.column-container {
+  display: grid;
+  grid-template-areas:
+    "select-dates-intro"
+    "select-dates"
+    "selected-dates-intro"
+    "selected-dates";
+
+  .select-dates-intro {
+    grid-area: select-dates-intro;
+  }
+
+  .select-dates {
+    grid-area: select-dates;
+  }
+
+  .selected-dates-intro {
+    grid-area: selected-dates-intro;
+  }
+
+  .selected-dates {
+    grid-area: selected-dates;
+  }
+
+  // show lists side-by-side on wider screens
+  @include breakpoint(md) {
+    grid-template-columns: 45% 1fr;
+    grid-template-areas:
+      "select-dates-intro selected-dates-intro"
+      "select-dates selected-dates";
+  }
+}
+
+// expandable info panel
+.dates-info {
+  background: $blue-active-lighter;
+
+  .expand-header {
+    cursor: pointer;
+  }
+
+  .expand-content {
+    .step {
+      position: relative;
+
+      &:not(:last-of-type) {
+        border-left: 1px solid $grey-medium;
+      }
+
+      padding-left: 1.8em;
+      padding-bottom: 1em;
+      margin-left: 1em;
+
+      // circled number icon
+      .number {
+        $icon-size: 2em;
+        $font-size: 1.2em;
+
+        font-size: $font-size;
+        position: absolute;
+        top: (1em - $font-size) * 0.5;
+        left: -$icon-size * 0.5;
```

Comment:
Fun fact, the `/` character is used as a separator in CSS4, so Sass is phasing out "/" for math division. The version we're using doesn't support the `div()` method we're supposed to use instead, so I'm just multiplying by 0.5 instead of dividing by 2 to avoid the whole thing and keep it forward compatible